experimental/Schemes/BlowupMorphism.jl

```diff
@@ -278,3 +278,100 @@ function maps_with_given_codomain(phi::CoveringMorphism, V::AbsSpec)
+
+julia> I = ideal(R,[x,y,z]);
+
+julia> bl = blow_up(A3,I)
+BlowupMorphism{CoveredScheme{QQField}}(relative projective scheme over covered scheme with 1 affine patches in its default covering, covered scheme with 1 affine patches in its default covering, sheaf of ideals on covered scheme with 1 affine patches in its default covering, #undef, #undef, Oscar.EffectiveCartierDivisor{CoveredScheme{QQField}}(covered scheme with 3 affine patches in its default covering, sheaf of ideals on covered scheme with 3 affine patches in its default covering, Covering with 3 patches, #undef))
+
+julia> show_details(bl)
+Blow up of a Covered Scheme with 1 Charts at a 0-dimensional Center leading to a Covered Scheme with 3 Charts.
+
+
+julia> show_details(bl;more=true)
+Blow up of a Covered Scheme with 1 Charts at a 0-dimensional Center leading to a Covered Scheme with 3 Charts.
+
+=====================================
+Affine charts of the original scheme:
+Spec of Multivariate Polynomial Ring in x, y, z over Rational Field
+
+=====================================
+Affine charts of the blown up scheme:
+Spec of Localization of Quotient of Multivariate Polynomial Ring in (s1//s0), (s2//s0), x, y, z over Rational Field by ideal(-(s1//s0)*z + (s2//s0)*y, (s2//s0)*x - z, (s1//s0)*x - y) at the multiplicative set powers of QQMPolyRingElem[1]
+
+Spec of Localization of Quotient of Multivariate Polynomial Ring in (s0//s1), (s2//s1), x, y, z over Rational Field by ideal((s2//s1)*y - z, -(s0//s1)*z + (s2//s1)*x, -(s0//s1)*y + x) at the multiplicative set powers of QQMPolyRingElem[1]
+
+Spec of Localization of Quotient of Multivariate Polynomial Ring in (s0//s2), (s1//s2), x, y, z over Rational Field by ideal(-(s1//s2)*z + y, -(s0//s2)*z + x, -(s0//s2)*y + (s1//s2)*x) at the multiplicative set powers of QQMPolyRingElem[1]
+
+=====================================
+Data of center:
+Ideal Sheaf on Covered Scheme with 1 Charts:
+
+Chart 1:
+        ideal(x, y, z)
+
+=====================================
+Exceptional divisor:
+Effective Cartier Divisor on Covered Scheme with 3 Charts:
+
+Chart 1:
+        ideal in Localization of Quotient of Multivariate Polynomial Ring in (s1//s0), (s2//s0), x, y, z over Rational Field by ideal(-(s1//s0)*z + (s2//s0)*y, (s2//s0)*x - z, (s1//s0)*x - y) at the multiplicative set powers of QQMPolyRingElem[1] generated by [x]
+
+Chart 2:
+        ideal in Localization of Quotient of Multivariate Polynomial Ring in (s0//s1), (s2//s1), x, y, z over Rational Field by ideal((s2//s1)*y - z, -(s0//s1)*z + (s2//s1)*x, -(s0//s1)*y + x) at the multiplicative set powers of QQMPolyRingElem[1] generated by [y]
+
+Chart 3:
+        ideal in Localization of Quotient of Multivariate Polynomial Ring in (s0//s2), (s1//s2), x, y, z over Rational Field by ideal(-(s1//s2)*z + y, -(s0//s2)*z + x, -(s0//s2)*y + (s1//s2)*x) at the multiplicative set powers of QQMPolyRingElem[1] generated by [z]
+```
+"""
+function show_details(Bl::BlowupMorphism; more::Bool = false)
```

Comment:
Maybe consider adding an optional `io::IO` argument, just like `show`, so that one can conveniently print this data into strings, files, etc.
```suggestion
function show_details(Bl::BlowupMorphism; more::Bool = false)
  show_details(stdout, Bl; more)
end

function show_details(io::IO, Bl::BlowupMorphism; more::Bool = false)
```
and then of course change all calls to `println` to `println(io, ...)`

Reply:
thank you, optional argument added as suggested.